pkgs/by-name/gr/granian/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  rustPlatform,
+  python312Packages,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "granian";
+  version = "1.5.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "emmett-framework";
+    repo = pname;
```

```suggestion
    repo = "granian";
```

`pname` in URLs is discouraged, and it's overused elsewhere as well